rectCameraMatrix:
  - 6.6760052591980673e+02
  - 6.6702754763359098e+02
  - 1.2238794285085744e+03
  - 9.1994665573314273e+02
handeyeTransform:
  # - -0.04123296
  # - 0.00050579
  # - 0.04795275
  # - -0.13260504 
  # - 0.13587395  
  # - -0.69586033 
  # - 0.6926273
  - -0.04196641  
  - 0.0012018   
  - 0.05935809
  - -0.14216268 
  - 0.1234418   
  - -0.69520724 
  - 0.69371377
laserPlane:
  # - -5.03368
  # - -103.61
  # - -100
  # - 11.1094
  - -3.10622
  - -102.576
  - -100
  - 11.1776


# laser Detector params

ROI_x: 350
ROI_y: 450
ROI_width: 1630
# ROI_height: 830
ROI_height: 1200 
img_width: 2448
img_height: 1840
hue_min: 10
hue_max: 30
sat_min: 150
val_min: 200
val_ratio: 0.05
f_vis: False 

cameraTopic: "/ximea_cam/image_raw/compressed"  # Got rid of the visual republish
statusTopic: "/execution_status"
pcTopic: "/laser_point_cloud"

